packages/datagateway-dataview/src/__mocks__/loglevel.ts

```diff
@@ -0,0 +1,3 @@
+// TODO: move __mocks__ folder back to package root once facebook/create-react-app#7539 is fixed
+
+export default jest.createMockFromModule('loglevel');
```

Comment:
I would assume this is to avoid excessive warning/error logs in the console? 😆

Reply:
This is to avoid the errors from "handleICATError" that are triggered when certain tests test failing request behaviour. Note I'm not mocking `console.error` so any actual errors from other libraries etc. should come through. Also, you can test that `log.error` etc. is called in your tests if you want (and some do) to test that errors are being logged - just not dirtying up the console!

Reply:
I see. I agree with mocking it globally as it makes the most sense to me as well.